Contributing/How-to-Assign-a-Version.md

```diff
@@ -1,34 +1,33 @@
 ## moja global Versioning
 
-Most moja global tools are platforms combined with modules. This results in many dependencies which in turn require tight version control without losing flexibility. 
+Most moja global tools are platforms combined with modules. This results in many dependencies which requires tight version control without losing any flexibility. 
 
-moja global projects use Vincent Driessen's [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html) and therefore moja global uses the [related versioning system](https://datasift.github.io/gitflow/Versioning.html). More eleborate discussion can be found [in the on-line discussion of SemVer](https://github.com/semver/semver/issues/323).  
+moja global projects use Vincent Driessen's [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html) and therefore moja global uses the [related versioning system](https://datasift.github.io/gitflow/Versioning.html). You can find a more elaborate discussion on the following [discussion of SemVer](https://github.com/semver/semver/issues/323).  
 
-The Standard Format for the version number consists of 3 indicators (MAJOR.MINOR.PATCH) and is only assigned at the moment the version is released. For work in progress a label is used as a placeholder.  
+The Standard Format for the version number consists of 3 indicators: **MAJOR.MINOR.PATCH**. It is only assigned at the moment the version is released. For a work in progress, a label is used as a placeholder.  
 
-### Standard Format  
+### Standard Format
```

Comment:
I'd forgotten about this, but we're aiming for a 6mo release cycle. My addition of a changelog has been left languishing. We should endeavour to update and publish a new release as soon as possible.

https://github.com/moja-global/FLINT/pull/71/files

Reply:
I will make a note for that and make sure that its discussed in a recent meeting.